design-library/source/_patterns/03-organisms/02-header/header.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Please follow the conventions i used in the other patters, which captures pattern dependencies

Reply:
Indeed; done!